js/mobile.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-plusplus */
```

Comment:
- [ ] Let's not disable linter rules. These are here to write us best code possible,